deploy_dod_dataset.py

```diff
@@ -1,10 +1,17 @@
 from io import BytesIO
 import boto3
 import os
+import logging
 
+from libs.build_params import OUTPUT_DIR_DOD
+from libs.constants import INTERVENTIONS
+from libs.validate_results import validate_states_df, validate_counties_df, validate_states_shapefile, validate_counties_shapefile
 from libs.build_dod_dataset import get_usa_by_county_df, get_usa_by_states_df, get_usa_county_shapefile, get_usa_state_shapefile
 from libs.build_dod_dataset import get_usa_by_county_with_projection_df
 
+logger = logging.getLogger()
```

Comment:
`logger = logging.getLogger(__name__)` is preferred since it'll prepend the logs with the module name so you know where it came from